src/batch-client/index.ts

```diff
@@ -822,6 +822,15 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(normalize(SearchResponse));
 
+	public sendDeliveryReport = (messageId: string) => {
+		return this.jsonRequest({
+			name: 'SendDeliveryReport',
+			body: {
+				mid: messageId
```

Comment:
this conversion should happen using denormalization

Reply:
Done.